common/io/io.cpp

```diff
@@ -235,10 +235,10 @@ ExpectedSharedIfstream OpenSharedIfstream(const string &path) {
 	return make_shared<ifstream>(std::move(exp_is.value()));
 }
 
-ExpectedOfstream OpenOfstream(const string &path) {
+ExpectedOfstream OpenOfstream(const string &path, bool append) {
 	ofstream os;
 	errno = 0;
-	os.open(path);
+	os.open(path, append ? ios::app : ios::out);
```

Comment:
Doesn't this need to be `append ? (ios::app | ios::out) : ios::out`?

Reply:
Not sure, the docs don't say. But it works with `ios::app` alone.